.meta/transforms/remap.toml.erb

```diff
@@ -0,0 +1,113 @@
+[transforms.remap]
+title = "Remap"
+allow_you_to_description = "remap one or more log fields"
+beta = false
+common = false
```

Comment:
I think it's safe to say this will be a common transform one it leaves beta (https://github.com/timberio/vector/pull/3341/files#r471660149)